docs/core-protocol/smart-contracts-api/PublicLock.md

```diff
@@ -1579,22 +1548,6 @@ function unlockProtocol() external view returns (address)
 |---|---|---|
 | _0 | address | undefined |
 
-### updateBeneficiary
-
-```solidity
-function updateBeneficiary(address _beneficiary) external nonpayable
```

Comment:
Are we really removing this function?

Reply:
Yes.. It was kind of redundant and we need to reduce the size. The `withdraw` function now takes a beneficiary address which can be used for these!